### DISCUSSION TITLE
Release 1.0.0

### DISCUSSION THREAD

**Discussion by vlsi** [robstoll/atrium#1349] [Ideas]
The current Atrium is usable, so it qualifies for 1.0.0

WDYT?

**Comment by robstoll** [comment]
we will drop support for kotlin 1.2 with the release of 0.20.0 maybe a good point in time to release 1.0.0 instead of 0.20.0

but to be honest, we should have released 1.0.0 with 0.18.0, so maybe it's even OK to release it instead of 0.19.0 and drop support for kotlin 1.2 with 1.1.0

that being said, maybe we should release 1.0.0 based 0.18.0. 0.19.0 will become 1.1.0 and 0.20.0 1.2.0

well, waiting for 0.19.0 makes sense because we will change artifact names due to the switch to the new MPP plugin and I will release it this month.
And then I land at 0.20.0 again as we drop translation support in this version. But there are always things I want to improve, especially on the logic/core level which are only possible with breaking changes.

I will release 0.19.0 as 1.0.0, and branch away for all breaking changes into a v2 branch.
